File: Java/Peripheral.java
// Peripheral.java
public abstract class Peripheral extends Elektronik {
    protected String tipePeripheral;
    protected String koneksi;
    
    public Peripheral() {
        super();
        this.tipePeripheral = "";
        this.koneksi = "";
    }
    
    public Peripheral(String merk, String nama, double harga, String tipePeripheral, String koneksi) {
        super(merk, nama, harga);
        this.tipePeripheral = tipePeripheral;
        this.koneksi = koneksi;
    }
    
    public String getTipePeripheral() {
        return tipePeripheral;
    }
    
    public void setTipePeripheral(String tipePeripheral) {
        this.tipePeripheral = tipePeripheral;
    }
    
    public String getKoneksi() {
        return koneksi;
    }
    
    public void setKoneksi(String koneksi) {
        this.koneksi = koneksi;
    }
    
    @Override
    public void tampilkanInfo() {
        System.out.println(tipePeripheral + ": " + merk + " " + nama + 
                           " (Koneksi: " + koneksi + ")");
        System.out.println("    Harga: Rp" + harga);
    }
}

File: Python/Peripheral.py
# peripheral.py
from Elektronik import Elektronik

class Peripheral(Elektronik):
    """Second branch of hierarchical inheritance - Peripherals"""
    
    def __init__(self, merk="", nama="", harga=0, tipe_peripheral="", koneksi=""):
        super().__init__(merk, nama, harga)
        self.tipe_peripheral = tipe_peripheral
        self.koneksi = koneksi
    
    def get_tipe_peripheral(self):
        return self.tipe_peripheral
    
    def set_tipe_peripheral(self, tipe_peripheral):
        self.tipe_peripheral = tipe_peripheral
    
    def get_koneksi(self):
        return self.koneksi
    
    def set_koneksi(self, koneksi):
        self.koneksi = koneksi
    
    def tampilkan_info(self):
        print(f"{self.tipe_peripheral}: {self.merk} {self.nama} (Koneksi: {self.koneksi})")
        print(f"    Harga: Rp{self.harga}")